modules/interacting-serverless-apps-http2-gRPC.adoc

```diff
@@ -15,7 +15,7 @@ To do this you must find the ingress gateway's public address and the applicatio
 . The ingress gateway's public address can be determined using this command:
 +
 ----
-oc -n istio-system get svc istio-ingressgateway
+$ oc -n knative-serving-ingress get svc kourier
 ----
 +
 The output will be similar to this example:
```

Comment:
We also need to change the output of this example. It should be something like this:
```
----
NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP                                                             PORT(S)                                                                                                                                      AGE
kourier   LoadBalancer   172.30.51.103   a83e86291bcdd11e993af02b7a65e514-33544245.us-east-1.elb.amazonaws.com   80:31380/TCP,443:31390/TCP   67m
----
```